pkg/arm/v7/resources.go

```diff
@@ -365,6 +365,17 @@ func vmss(cs *api.OpenShiftManagedCluster, app *api.AgentPoolProfile, backupBlob
 		script = base64.StdEncoding.EncodeToString(b)
 	}
 
+	vmssTags := map[string]*string{
+		"clusterName": to.StringPtr(cs.Name),
+		"logAnalyticsWorkspaceResourceId": to.StringPtr(resourceid.ResourceID(
+			cs.Properties.AzProfile.SubscriptionID,
+			cs.Properties.AzProfile.ResourceGroup,
+			"Microsoft.OperationalInsights/workspaces",
+			cs.Properties.MonitorProfile.WorkspaceID)),
```

Comment:
@charlesakalugwu this might be nice, but it is causing all nodes to be rotated if you enable/disable monitoring. Can we do this is a less intrusive way?

Reply:
I have just pushed a commit to remove it, let me know what you think..